### ISSUE TITLE
bug: wp plugin install renames plugin to 'archive'

### ISSUE THREAD

**Issue by scarstens** [wp-cli/wp-cli#4620]
I'm using WP CLI to install woo-sensei from the open source repo, but its doing something weird on the rename. Try the following:
```bash
wp plugin install "https://github.com/Automattic/sensei/archive/version/1.9.19.zip" --force
```
which results in
```
Downloading installation package from https://github.com/Automattic/sensei/archive/version/1.9.19.zip...
Unpacking the package...
Installing the plugin...
Renamed Github-based project from 'sensei-version-1.9.19' to 'archive'.
Removing the old version of the plugin...
Plugin updated successfully.
Success: Installed 1 of 1 plugins.
```

My current workaround is to pull down the master version, as it figure out how to rename that one without a problem. Here is what that looks like (this is working fine):
```bash
wp plugin install "https://github.com/Automattic/sensei/archive/master.zip" --force
```
Results:
```
Downloading installation package from https://github.com/Automattic/sensei/archive/master.zip...
Unpacking the package...
Installing the plugin...
Renamed Github-based project from 'sensei-master' to 'sensei'.
Removing the old version of the plugin...
Plugin updated successfully.
Success: Installed 1 of 1 plugins.
```

At this point, I'm unblocked, but I'd rather keep my provision script limited to a tagged version so some random update doesn't break my site. Any thoughts?

**Comment by gitlost** [comment]
Closing in favor of https://github.com/wp-cli/extension-command/issues/80
